include/DREAM/OtherQuantityHandler.hpp

```diff
@@ -21,11 +25,21 @@ namespace DREAM {
         CollisionQuantityHandler *cqtyHottail, *cqtyRunaway;
         PostProcessor *postProcessor;
         RunawayFluid *REFluid;
+        FVM::UnknownQuantityHandler *unknowns;
+        std::vector<UnknownQuantityEquation*> *unknown_equations;
+        Settings *s;
         FVM::Grid *fluidGrid, *hottailGrid, *runawayGrid;
+
+        len_t id_Tcold;
+        len_t id_ncold;
+        UnknownQuantityEquation *eqn_Tcold;
+        len_t id_term_rad;
+        
+
     public:
         OtherQuantityHandler(
             CollisionQuantityHandler*, CollisionQuantityHandler*,
-            PostProcessor*, RunawayFluid*,
+            PostProcessor*, RunawayFluid*, FVM::UnknownQuantityHandler*, std::vector<UnknownQuantityEquation*>*, Settings*,
```

Comment:
I thought it convenient to not have to add more input arguments whenever one wants to add another term to the output, so this "error" is actually deliberate. Is there a good way to have access to all information about which terms are actually included in the OtherQuantityHandler or is adding more input arguments the best solution anyway?

Reply:
I actually favour adding more input arguments. Alternatively, we could add an enum to the ``OtherQuantityHandler`` containing all the options we need to store. Just inserting a ``Settings`` object is not very verbose (it's unclear how it will actually be used), and it's not how it was intended to be used.

Reply:
(another related reason is that we should try to limit the amount of hard-coding of settings names. By keeping the ``Settings`` object to the ``SimulationGenerator`` we can keep all that hard-coding to files under ``src/Settings/``)